pkg/ingester/ingester_v2.go

```diff
@@ -390,6 +390,85 @@ func (i *Ingester) v2MetricsForLabelMatchers(ctx old_ctx.Context, req *client.Me
 	return result, nil
 }
 
+// QueryTSDBStream streams metrics from a TSDB. This implements the client.IngesterServer interface
+func (i *Ingester) QueryTSDBStream(req *client.QueryRequest, stream client.Ingester_QueryStreamServer) error {
```

Comment:
So far we've prefixed the TSDB functions as `v2<name>`, so I would get stick to it for consistency (`v2QueryStream()`). I'm not a big fan of `v2` at all, but we can do a mass renaming in a dedicated PR.

Reply:
Yea that sounds like good consistency